analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/SymbolicExecution/SETestContext.cs

```diff
@@ -88,6 +88,9 @@ End Sub
     Private Shared Sub Tag(Name As String, Optional Arg As Object = Nothing)
     End Sub
 
+    Private Shared Function Unknown(Of T)() As T
+    End Function
```

Comment:
I think this is unused now. 

You can remove it. Or keep it, it might be handy in the future. Up to you.

Reply:
I'll leave it to be in sync with the C# version.